examples/connext_dds/group_coherent_presentation/py/GroupCoherentExample_subscriber.py

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
I don't see `print_coherent_set_info` either. Review the C++ and C# examples. It looks like the logic is not the same in Python.

Reply:
Also, run the example in C++ or C#, review the output and compare it to the ../README.md's Example Description and make sure you see a similar output and behavior in python.